test/lolex-test.js

```diff
@@ -114,6 +114,50 @@ describe("issue sinon#1852", function () {
     });
 });
 
+describe.only("issue #207 - nanosecond round-off errors on high-res timer", function () {
```

Comment:
That's an exclusive test. If we land on this commit using `git bisect`, we might mistakenly think all is well. Will you remove this from history, or are you planning to use "Squash and merge" when merging this PR?

Reply:
Didn't see that. I thought this would make the most sense as a merge commit. Can try to fix it.